packages/neon-core/__integration__/rpc/RPCClient.ts

```diff
@@ -12,7 +12,7 @@ const TESTNET_URLS = [
 ];
 
 let client: rpc.RPCClient;
-const address = "ALq7AWrhAueN6mJNqk6FHJjnsEoPRytLdW";
+const address = "AXSoNQEKjmqPBNPg5cNrHyWivfjok3Vj9D";

```

Comment:
Is this change necessary?

Reply:
yes, because the other address doesn't have a balance

Reply:
What balance are you looking at? So I mainly use that address since the private key is available in the repository for use. It would be simpler if we can continue to use the test keys instead of a custom address that the repository does not control for integration tests.

Reply:
I really don't need to change the address, I'll go back to the other address